recipe/build.sh

```diff
@@ -1,45 +1,37 @@
 #!/bin/bash
 
-mkdir build
+mkdir build || true
 cd build
 
-
 declare -a CMAKE_PLATFORM_FLAGS
 if [[ ${target_platform} =~ osx.* ]]; then
     # only needed until conda-forge toolchain 2.3.0 is used
     #   https://github.com/conda-forge/toolchain-feedstock/pull/47
     CMAKE_PLATFORM_FLAGS+=(-DCMAKE_OSX_SYSROOT="${CONDA_BUILD_SYSROOT}")
-
-    # add rpath for executables
-    export LDFLAGS="${LDFLAGS} -Wl,-rpath,${PREFIX}/lib"
-else
+elif [[ ${target_platform} =~ linux.* ]]; then
     CMAKE_PLATFORM_FLAGS+=(-DCMAKE_TOOLCHAIN_FILE="${RECIPE_DIR}/cross-linux.cmake")
     # link transitive ADIOS1 libraries during build of intermediate wrapper lib
     export LDFLAGS="${LDFLAGS} -Wl,-rpath-link,${PREFIX}/lib"
 fi
 
-export LD_LIBRARY_PATH="${PREFIX}/lib:$LD_LIBRARY_PATH"
+bash ${SRC_DIR}/.travis/download_samples.sh
 
 cmake \
-    -DCMAKE_BUILD_TYPE=Release  \
-    -DopenPMD_USE_MPI=OFF       \
-    -DopenPMD_USE_HDF5=ON       \
-    -DopenPMD_USE_ADIOS1=ON     \
-    -DopenPMD_USE_ADIOS2=OFF    \
-    -DopenPMD_USE_PYTHON=ON     \
-    -DopenPMD_USE_INTERNAL_PYBIND11=OFF              \
+    -DCMAKE_BUILD_TYPE=Release                       \
+    -DCMAKE_SYSTEM_IGNORE_PATH=/usr/lib              \
+    -DCMAKE_INSTALL_LIBDIR=lib                       \
```

Comment:
that should be the default: https://github.com/openPMD/openPMD-api/blob/4cbe73c1f824b6d8509131fffe6adc20f7670cdc/CMakeLists.txt#L23

Reply:
Explicit is better than implicit, but I do not see anything setting DCMAKE_SYSTEM_IGNORE_PATH which is critical to prevent it finding system zlib and bzip2

Reply:
Explicit is better than implicit, but I do not see anything setting DCMAKE_SYSTEM_IGNORE_PATH which is critical to prevent it finding system zlib and bzip2